api/src/main/java/gov/cms/ab2d/api/config/CorsConfig.java

```diff
@@ -20,7 +20,7 @@ public FilterRegistrationBean corsFilter() {
         UrlBasedCorsConfigurationSource source = new UrlBasedCorsConfigurationSource();
         CorsConfiguration config = new CorsConfiguration();
         config.setAllowCredentials(true);
-        config.addAllowedOrigin("*");
+        config.addAllowedOrigin("127.0.0.1:4000");
```

Comment:
Shouldn't this be `ab2d.cms.gov`?

Reply:
We could make this yet another configuration property via an env variable

Reply:
I think when it's deployed there it would be coming from the same origin, when it's running locally the site runs on port 4000, and the app is on port 8080. Unless there's different origins between the app and website when it's deployed?

Reply:
When initiated via the Web site, API requests would show origin of `https://ab2d.cms.gov`, which is still different from the API subdomain, so it will likely fail. We should probably make this an environment variable-driven property just like everything else, and use `https://ab2d.cms.gov` as the default value, while CI & local envs could override as needed.